src/orquestra/sdk/_ray/_build_workflow.py

```diff
@@ -3,9 +3,12 @@
 ################################################################################
 """Translates IR workflow def into a Ray workflow."""
 import os
+import re
 import time
 import typing as t
+import warnings
 from functools import singledispatch
+from importlib.metadata import version
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏We already use expose this https://github.com/zapatacomputing/orquestra-workflow-sdk/blob/main/src/orquestra/sdk/packaging/__init__.py#L16
```suggestion
from orquestra.sdk.packaging import get_installed_version
```